atomistics/workflows/quasiharmonic/workflow.py

```diff
@@ -33,8 +34,17 @@ def __init__(
         primitive_matrix=None,
         number_of_snapshots=None,
     ):
+        repeat_vector = np.array(
+            np.diag(
+                get_supercell_matrix(
+                    interaction_range=interaction_range,
+                    cell=structure.cell.array,
+                )
+            ),
+            dtype=int,
+        )
```

Comment:
I would like an explanation of intent, although it doesn't need to be in this place or take this exact form

```suggestion
        )
        # Phonopy internally repeats structures that are "too small"
        # Here we manually guarantee that all structures passed are big enough
        # This provides some computational efficiency for classical calculations
        # And for quantum calculations _ensures_ that force matrices and energy/atom
        # get treated with the same kmesh
```